spec/ruby_ami/stream_spec.rb

```diff
@@ -175,6 +175,26 @@ def mocked_server(times = nil, fake_client = nil, &block)
       ]
     end
 
+    it 'sends events with multiple headers with the same key to the client when the stream is ready' do
+      mocked_server(1, lambda { @stream.send_data 'Foo' }) do |val, server|
+        server.send_data <<-EVENT
+Event: Status
+Channel: SIP/101-3f3f
+Variable: CDR(ds_type)=Test-INT
+Variable: from_user=ssether
+Variable: companyID=Test
+ActionID: 2
+
+        EVENT
+      end
+
+      client_messages.should be == [
+        [Stream::Connected.new, @stream],
+        [Event.new('Status', 'Channel' => 'SIP/101-3f3f', 'Variable' => 'CDR(ds_type)=Test-INT,from_user=ssether,companyID=Test', 'ActionID' => '2'), @stream],
```

Comment:
What about this?

```suggestion
        [Event.new('Status', 'Channel' => 'SIP/101-3f3f', 'Variable' => ['CDR(ds_type)=Test-INT', 'from_user=ssether', 'companyID=Test'], 'ActionID' => '2'), @stream],
```

Reply:
I'd rather a 'string' to not change the interface, but it could return a 'map' with key / values

Reply:
The problem with a Hash is that it is specific to the `Variable` header, and not a generic way of expressing multiple values for any header.